main.tf

```diff
@@ -254,13 +249,13 @@ resource "ibm_database" "postgresql_db" {
     content {
       group_id = "member" # Only member type is allowed for IBM Cloud Databases
       memory {
-        allocation_mb = var.member_memory_mb
+        allocation_mb = var.memory_mb
       }
       disk {
-        allocation_mb = var.member_disk_mb
+        allocation_mb = var.disk_mb
       }
       cpu {
-        allocation_count = var.member_cpu_count
+        allocation_count = var.cpu_count
       }
       dynamic "members" {
```

Comment:
add ` #  There is only 1 operation type for PostgreSQL so it is not exposed as a configuration`

<img width="1647" alt="image" src="https://github.com/user-attachments/assets/a05b6b46-d399-47f0-98e0-84f2833ab310" />